app/routes/index.tsx

```diff
@@ -254,40 +271,71 @@ export default function Index() {
                         </Field>
 
                         <Field name="loginRedirect" />
-                        <div className="mt-8 mb-4">
-                          <button
-                            type="submit"
-                            className="btn btn-primary w-full"
-                          >
+                        <div className="mt-4 mb-2">
+                          <Button size="large" fullSize>
                             Anmelden
-                          </button>
+                          </Button>
                         </div>
                       </>
                     )}
                   </LoginForm>
-
-                  <div className="mb-8 text-center">
-                    <Link
-                      to={`/reset${
-                        loginRedirect ? `?login_redirect=${loginRedirect}` : ""
-                      }`}
-                      className="text-primary font-bold underline"
-                    >
-                      Passwort vergessen?
-                    </Link>
-                  </div>
-
-                  <div className="text-center">
-                    Noch kein Mitglied?{" "}
-                    <Link
-                      to={`/register${
-                        loginRedirect ? `?login_redirect=${loginRedirect}` : ""
-                      }`}
-                      className="text-primary font-bold underline"
-                    >
-                      Registrieren
-                    </Link>
-                  </div>
+                  {loaderData.abilities.keycloak.hasAccess ? (
+                    <>
+                      <div className="text-center">Noch kein Mitglied?</div>
+                      <div className="flex justify-center gap-6">
+                        <Link
+                          to={`/register${
+                            loginRedirect
+                              ? `?login_redirect=${loginRedirect}`
+                              : ""
+                          }`}
+                          className="text-primary font-semibold underline"
+                        >
+                          Registrieren mit E-Mail
+                        </Link>
+
+                        <Link
+                          to={`/auth/keycloak"${
```

Comment:
I think a double quote sneaked in here. Or is this intentional?

Reply:
🔎